infra/cloudflare-r2-wrangler/blockprotocol-preview/wrangler.toml

```diff
@@ -1,10 +1,10 @@
-name = "blocks-dev"
+name = "blockprotocol-dev"
 main = "../index.js"
 compatibility_date = "2022-06-30"
 
 account_id = "c6499786332a3d2fb35419a7803ab7aa"
 workers_dev = true
 
 [[r2_buckets]]
-binding = "BLOCK_BUCKET"
-bucket_name = "blocks-dev"
\ No newline at end of file
+binding = "BLOCKPROTOCOL_BUCKET"
+bucket_name = "blockprotocol-preview"
```

Comment:
We say `-preview` but call the wrangler `-dev` – can we make it consistently `dev`, `preview` or `staging`? I like `preview` least, I think, but don't feel strongly about it.

Reply:
Fair point! I changed `name` to `blockprotocol-preview` in https://github.com/blockprotocol/blockprotocol/pull/565/commits/af50f35f73f3ea7d97983d3f5e0347e82b7fb934. I get what you mean by this word being far from ideal, but I see benefits in using a consistent term with Vercel. If preview/production releases match preview/production resource suffix, there is less room for confusion. We can't change ids of Vercel environments, so we can just mirror them. I also renamed production wranger in https://github.com/blockprotocol/blockprotocol/pull/565/commits/9aa32892c12223f63e54f46067f55754e94c4e49 for consistency.

Happy to change names & suffixes to something else if there are arguments stronger than Vercel consistency.

Reply:
I thought I had made this comment as part of the review, but think I left it in draft or something – the other consideration for production is that the URL to the assets is driven by the wrangler name, which is driven by the `name` field. 

For production, could we use simply `blockprotocol` as the wrangler name? that will give us `blockprotocol.hashai.workers.dev/path` urls rather than `blockprotocol-production.hashai.workers.dev/path` urls

the bucket name can stay the same or change to lose the suffix, doesn't affect the URLS

Reply:
I see, makes sense. I renamed the production wrangler in https://github.com/blockprotocol/blockprotocol/pull/565/commits/390dddd4d5fd8b79fc393ff3523c68cc83cccf67. Did I do this correctly?

Reply:
I see, makes sense. I renamed the production wrangler in https://github.com/blockprotocol/blockprotocol/pull/565/commits/390dddd4d5fd8b79fc393ff3523c68cc83cccf67. Did I do this correctly?